test/acceptance/p2p/muxer.cpp

```diff
@@ -96,6 +99,15 @@ struct Server : public std::enable_shared_from_this<Server> {
 
     stream->readSome(
         *buf, buf->size(), [buf, stream, this](outcome::result<size_t> rread) {
+          if (!rread) {
+            if (rread.error() == YamuxedConnection::Error::CLOSED_BY_PEER) {
+              // TODO(artem): EOF wasn't signalled by former logic,
```

Comment:
Please leave Jira task id to make it possible to track the debt, otherwise - we have to fix this before the merge.